Cargo.toml

```diff
@@ -5,6 +5,18 @@ version = "1.0.0"
 authors = ["Brett Cannon <brett@python.org>"]
 repository = "https://github.com/brettcannon/python-launcher"
 readme = "README.md"
+include = [
+    "Cargo.toml",
+    "/src/",
+    "/benches/",
+    "/tests/",
```

Comment:
I don't include these in the tarballs, so I'm wondering why you're suggesting to keep them here? In case someone wants to verify their build works and see how fast it is compared to a previous build?

Reply:
Alright, including benchmarks is maybe a bit much.

But including integration tests would be great, since we're running the whole test suite for all our Rust packages during package builds, where at all possible. This lets us be more confident in actually shipping working software, on many architectures (x86, ARM, PowerPC, System Z, etc.), and lets us verify to some degree that the modifications we need to make to some crates do not break anything.

Reply:
I figured about the testing bit, so all good for me to keep it. And keeping `benches/` so that everything specified in `Cargo.toml` still builds without issue is also fine with me!